fe/fe-core/src/main/java/org/apache/doris/nereids/stats/StatsCalculator.java

```diff
@@ -649,6 +649,11 @@ private Statistics computeCatalogRelation(CatalogRelation catalogRelation) {
         TableIf table = catalogRelation.getTable();
         double rowCount = catalogRelation.getTable().estimatedRowCount();
         boolean hasUnknownCol = false;
+        long idxId = -1;
+        if (catalogRelation instanceof OlapScan) {
+            OlapScan olapScan = (OlapScan) catalogRelation;
+            idxId = olapScan.getSelectedIndexId();
```

Comment:
What if the selected index is base index? I think in this case, we need to set idxId back to -1. Something like:
```
if (idxId == ((OlapTable) table).getBaseIndexId()) {
idxId = -1
}
```

Reply:
done